### DISCUSSION TITLE
min-width Button

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1149] [Proposals]
### Goal

Ao definir o valor de min-width dos botões.

### Context

Para garantir que tenhamos uma largura mínima, mesmo que seja apenas uma palavra, min-width deve ser usado como uma boa prática. Nossa proposta de largura mínima é 100px. 

### Impact

https://ishadeed.com/article/min-max-css/

### Discussion Questions

Seria possível fazer esse ajuste?

### Do you have a visual proposal?

https://www.figma.com/file/seiNjy7YmPWEY3BfY4WYpp/Novos-componentes---Sprint-1?node-id=22%3A27735&t=gYOdlKhORV0w5rhF-1

**Comment by yceballost** [comment]
Genial, muchas gracias Anne! Lo incluimos para evolucionar

**Reply by annefranca** [reply]
Olá @yceballost :)

Essa tarefa ficaria com o seu time? Pergunto porque na planilha está como "VIVO".
[Componentes Matriz v3](https://telefonicacorp.sharepoint.com/:x:/s/Colabora_TCX/Ee9cFk_VD2NItAAsYdlrJLIB2bXcMKmXswLu7FC68oTcmg?e=1bBUrC)

**Reply by yceballost** [reply]
Lo hacéis vosotros!

He creado el ticket para que quede constancia del cambio.

https://github.com/Telefonica/mistica-design/issues/1154

**Reply by yceballost** [reply]
Una dúvida, el min-width que se cambiará a qué tipo de botón aplica? Default o Smalls?

**Reply by BrunoBiagioniNeto** [reply]
@yceballost Entonces hagamos el Diseño y Desarrollo en Brasil y seguiremos como un flujo de Desarrollo como los otros componentes, ¿correcto?

**Reply by annefranca** [reply]
@yceballost para o Small também, segue o link com a proposta:
[min-width Button](https://www.figma.com/file/seiNjy7YmPWEY3BfY4WYpp/Novos-componentes---Sprint-1?node-id=22%3A27735&t=H06qGuPEE3mjf1QF-1)

**Reply by yceballost** [reply]
branch creada para que puedas añadirlo a las specs:
https://www.figma.com/file/koROdh3HpEPG2O8jG52Emh/branch/PGr1qnVJ91Alkj14XJN7iI/%F0%9F%94%B8-Buttons-Component-Specs?node-id=0%3A1&t=TEH8iPWOv4ckpbX0-1
